src/imitation/scripts/analyze.py

```diff
@@ -196,34 +194,26 @@ def _return_summaries(sd: sacred_util.SacredDicts) -> dict:
     )
 
 
-sd_to_table_entry_type = Mapping[str, Callable[[sacred_util.SacredDicts], Any]]
+sd_to_table_entry_type = Dict[str, Callable[[sacred_util.SacredDicts], Any]]
```

Comment:
As before, we've usually used `Mapping` over `Dict` where we don't need `dict`-specific features